data/json/items/comestibles/irradiated_fruit.json

```diff
@@ -32,6 +32,17 @@
     "spoils_in": "0 hours",
     "container": "bag_plastic"
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "irradiated_juniper",
+    "copy-from": "juniper",
+    "name": "irradiated juniper",
+    "name_plural": "irradiated juniper berries",
+    "description": "An irradiated juniper berry will remain edible nearly forever.  Sterilized using radiation, so it's safe to eat.",
```

Comment:
Why would anyone irradiate a Juniper berry for preservation? The already last years, and shelf stability isn't a big concern for gin botanicals

Reply:
Mostly, it's just to make sure that this being a fruit type kind of item, and appearing among other fruits, that it has the same products like drying fruit and irradiated fruit. It doesn't have anything to do with the shelf life, just to put it next to others of its kind.